lib/dep-graph-builders/pnpm/utils.ts

```diff
@@ -34,15 +34,6 @@ export const getPnpmChildNode = (
     resolvedVersion = pkgData.version;
   }
   if (!pkgs[childNodeKey]) {
-    if (lockfileParser.isWorkspaceLockfile()) {
```

Comment:
This is not needed anymore because workspace importers are added to extracted packages and we should fail with OutOfSync if not found there